extensions/shared/register-jetpack-block.js

```diff
@@ -87,11 +86,16 @@ export default function registerJetpackBlock( name, settings, childBlocks = [] )
 		return false;
 	}
 
-	const result = registerBlockType( `jetpack/${ name }`, {
+	const jetpackBlockSettings = {
 		...settings,
 		title: buildBlockTitle( settings.title, buildBlockTags( name, requiredPlan ) ),
-		example: requiredPlan ? undefined : settings.example,
-	} );
+	};
+
+	if ( ! requiredPlan && settings.example ) {
+		jetpackBlockSettings.example = settings.example;
+	}
```

Comment:
Is this necessary? The `settings` object is spread as `jetpackBlockSettings` is initialised, so it will contain `example` if it exists.

Reply:
omg you caught me!! that's so silly 🙃 . Updating...

Reply:
Actually, I was right (the first time, not the second), because we'd like to add the example ONLY when the block doesn't require a paid plan.

Reply:
going to mark it as resolved

Reply:
Actually, we need to do the opposite. It means to remove the `example` key if the block requires a plan. I did here 4d098f2f9beec994fa96fdda1e753341309e3572, and it rolled back to the initial issue :-/

Reply:
updated: let's enable the block example preview for all blocks, all plans.